aviary/variable_info/variable_meta_data.py

```diff
@@ -203,6 +203,122 @@
+# |  _ \          | |   | |
+# | |_) |   __ _  | |_  | |_    ___   _ __   _   _
+# |  _ <   / _` | | __| | __|  / _ \ | '__| | | | |
+# | |_) | | (_| | | |_  | |_  |  __/ | |    | |_| |
+# |____/   \__,_|  \__|  \__|  \___| |_|     \__, |
+#                                             __/ |
+#                                            |___/
+# =================================================
+add_meta_data(
+    Aircraft.Battery.ADDITIONAL_MASS,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": 'aircraft.inputs.L0_battery.weight_offset'
+                     },
+    units='lbm',
+    desc='mass of non energy-storing parts of the battery',
+    default_value=0.0,
+)
+
+add_meta_data(
+    Aircraft.Battery.DISCHARGE_LIMIT,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": 'aircraft.inputs.L0_battery.depth_of_discharge'
+                     },
+    units='unitless',
+    desc='default constraint on how far the battery can discharge, as a percentage of '
+         'total energy capacity'
+)
+
+add_meta_data(
+    Aircraft.Battery.EFFICIENCY,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    units='unitless',
+    desc="battery pack efficiency"
+)
+
+add_meta_data(
+    Aircraft.Battery.ENERGY_CAPACITY,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    units='kJ',
+    desc="total energy the battery can store"
+)
+
+add_meta_data(
+    Aircraft.Battery.MASS,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": 'aircraft.inputs.L0_battery.weight'
+                     },
+    units='lbm',
+    desc='total mass of the battery',
+    default_value=0.0,
+)
+
+add_meta_data(
+    Aircraft.Battery.PACK_ENERGY_DENSITY,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": 'aircraft.inputs.L0_battery.energy_density'
+                     },
+    units='kW*h/kg',
+    desc='specific energy density of the battery pack',
+    default_value=1.0,
+)
+
+
+add_meta_data(
+    Aircraft.Battery.PACK_MASS,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    units='lbm',
+    desc='mass of the energy-storing components of the battery',
+    default_value=0.0,
+)
+
+add_meta_data(
+    Aircraft.Battery.PACK_VOLUMETRIC_DENSITY,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    units='kW*h/kg',
```

Comment:
Wh/L is the most common unit I've seen

Reply:
Yes, this currently has units for mass-specific energy. I don't have a preference for kW\*h vs. W*h for the numerator